src/commands/search.rs

```diff
@@ -43,7 +43,7 @@ fn run_search(directory: &Path, query: &str) -> tantivy::Result<()> {
         let mut scorer = weight.scorer(segment_reader)?;
         while scorer.advance() {
             let doc_id = scorer.doc();
-            let doc = segment_reader.doc(doc_id)?;
+            let doc = segment_reader.get_store_reader().get(doc_id)?;
```

Comment:
can you get the store reader outside of the loop?

Reply:
Sure! https://github.com/tantivy-search/tantivy-cli/pull/37/commits/7720853d4a37aec14411f49d9f7c283b34ee765f